docs/getting-started/install-elastic-endpoint.asciidoc

```diff
@@ -49,7 +49,7 @@ image::images/install-endpoint/filter-network-content.png[]
 [[enable-fda-endpoint]]
 == Enable Full Disk Access for {elastic-endpoint}
 
-{elastic-endpoint} requires Full Disk Access to subscribe to system events via the {elastic-defend} framework and to protect your network from malware and other cybersecurity threats. Full Disk Access permissions is a new privacy feature introduced in macOS Mojave (10.14) that prevents some applications from accessing your data. To enable Full Disk Access, you must manually approve {elastic-endpoint}. For endpoints running macOS Mojave (10.14) and earlier, you must also approve the {elastic-endpoint} <<kernel-extension-approval, kernel system extension>>.
+{elastic-endpoint} requires Full Disk Access to subscribe to system events via the {elastic-defend} framework and to protect your network from malware and other cybersecurity threats. Full Disk Access permissions is a new privacy feature introduced in macOS Mojave (10.14) that prevents some applications from accessing your data. To enable Full Disk Access on endpoints running macOS Catalina (10.15) and later, you must manually approve {elastic-endpoint}. For endpoints running macOS Mojave (10.14) and earlier, you must also approve the {elastic-endpoint} <<kernel-extension-approval, kernel extension>>.
```

Comment:
@Trinity2019 can you verify these changes when you have a chance? I wasn't sure how to update the versioning in the last sentence, so any input you have on that would be very much appreciated - thank you!

Reply:
I think for this sentence, the original one works:
> To enable Full Disk Access, you must manually approve {elastic-endpoint}.
The rest looks good. :)

Reply:
Thanks, @Trinity2019! I'll remove the last sentence (For endpoints running macOS Mojave (10.14) and earlier, you must also approve the {elastic-endpoint} <<kernel-extension-approval, kernel extension>>) when I make the changes for https://github.com/elastic/security-docs/issues/3477. 


```suggestion
{elastic-endpoint} requires Full Disk Access to subscribe to system events via the {elastic-defend} framework and to protect your network from malware and other cybersecurity threats. To enable Full Disk Access on endpoints running macOS Catalina (10.15) and later, you must manually approve {elastic-endpoint}. For endpoints running macOS Mojave (10.14) and earlier, you must also approve the {elastic-endpoint} <<kernel-extension-approval, kernel extension>>.
```